docs/src/drivers/pluto-p.adoc

```diff
@@ -41,23 +41,23 @@ http://wiki.linuxcnc.org/cgi-bin/wiki.pl?LinuxCNC_Supported_Hardware[LinuxCNC Su
   voltage thresholds. The pins are all configured in 'LVTTL/LVCMOS' mode
   and are generally compatible with 5V TTL logic.
 * Before configuration and after properly exiting LinuxCNC, all Pluto-P
-  pins are tristated with weak pull-ups (20k-ohms min, 50k-ohms max).
+  pins are tristated with weak pull-ups (20 kΩ min, 50 kΩ max).
   If the watchdog timer is enabled (the default),
   these pins are also tristated after an interruption of communication
   between LinuxCNC and the board. The watchdog timer takes approximately
-  6.5ms to activate. However, software bugs in the pluto_servo firmware
+  6.5 ms to activate. However, software bugs in the pluto_servo firmware
   or LinuxCNC can leave the Pluto-P pins in an undefined state.
 * In pwm+dir mode, by default dir is HIGH for negative values and LOW
   for positive values. To select HIGH for positive values and LOW for
   negative values, set the corresponding dout-NN-invert parameter TRUE
   to invert the signal.
 * The index input is triggered on the rising edge. Initial testing has
   shown that the QZx inputs are particularly noise sensitive, due to
-  being polled every 25ns. Digital filtering has been added to filter
-  pulses shorter than 175ns (seven polling times). Additional external
+  being polled every 25 ns. Digital filtering has been added to filter
+  pulses shorter than 175 ns (seven polling times). Additional external
   filtering on all input pins, such as a Schmitt buffer or inverter, RC
   filter, or differential receiver (if applicable) is recommended.
-* The IN1…IN7 pins have 22-ohm series resistors to their associated FPGA
+* The IN1…IN7 pins have 22Ω series resistors to their associated FPGA
```

Comment:
Should there be a space before Ω here?

Reply:
Yes, will add. https://tex.stackexchange.com/questions/20962/should-i-put-a-space-between-a-number-and-its-unit

I just wish that asciidoc could be a bit more expressive with the kerning. Style guide?